docs/content/Coding Milestones/PoC2023q1/example1-subs/example1-post-kcp.md

```diff
@@ -92,11 +92,66 @@ to pass the new file with `--kubeconfig` on the command lines when
 using kcp or KubeStellar.
 
 
+### Space Manager
+
+The Space Manager will use space provider to create and manage underlying cluster
```

Comment:
I thought we were making a distinction between:
- a "space provider", an underlying technology that implements multiple spaces and is independent of the space framework; examples include kcp, kind, kubeflex, rancher; vs.
- a "provider adapter", the code that makes the connection between the space framework (Space Manager) and a particular space provider, thus depending on both.